projects/material-css-vars/src/lib/material-css-vars.service.ts

```diff
@@ -96,13 +97,16 @@ export class MaterialCssVarsService {
   }
 
   setVariable(cssVarName: MaterialCssVariables, value: string) {
-    this._setStyle([{
-      name: cssVarName,
-      val: value,
-    }]);
+    this._setStyle([
+      {
```

Comment:
In general there are a lot of code style changes, which makes the PR a bit hard to read.